Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -1749,47 +1753,64 @@ public class TimeSlider: UIControl {
         if isObserving {
             removeObservers()
         }
-        map?.addObserver(self, forKeyPath: #keyPath(AGSMap.operationalLayers), options: [.new, .old], context: &TimeSlider.KVOContext)
-        scene?.addObserver(self, forKeyPath: #keyPath(AGSScene.operationalLayers), options: [.new, .old], context: &TimeSlider.KVOContext)
-        geoView?.addObserver(self, forKeyPath: #keyPath(AGSGeoView.timeExtent), options: .new, context: &TimeSlider.KVOContext)
+        
+        // Observe operationalLayers of map
+        mapLayersObservation = map?.observe(\.operationalLayers, options: [.new, .old], changeHandler: { [weak self] (map, change) in
+            //
+            // Make sure self is around
+            guard let strongSelf = self else {
+                return
+            }
+            
+            // Handle the change in operationalLayers
+            strongSelf.handleOperationalLayers(change: change)
+        })
+        
+        // Observe operationalLayers of map
+        sceneLayersObservation = scene?.observe(\.operationalLayers, options: [.new, .old], changeHandler: { [weak self] (scene, change) in
+            //
+            // Make sure self is around
+            guard let strongSelf = self else {
+                return
+            }
+            
+            // Handle the change in operationalLayers
+            strongSelf.handleOperationalLayers(change: change)
+        })
+        
+        timeExtentObservation = geoView?.observe(\.timeExtent, options: .new, changeHandler: { [weak self] (scene, change) in
```

Comment:
Neither closure parameter is being used, so you could replace them both with `_`: `(_, _) in`

Reply:
fixed